app/config/config.yml

```diff
@@ -19,8 +23,6 @@ framework:
   validation: { enable_annotations: true }
   templating:
     engines: ['twig']
-    assets_version_format: '%%s?v=%%s'
-    assets_version: 1
```

Comment:
als ge dit doet, moet dit trouwens ook in het deploy script aangepast worden, want hij bumped die assets_version automatisch tijdens het deployen.